### DISCUSSION TITLE
libuv/1.14.2-dev

### DISCUSSION THREAD

**Discussion by dyussekeyev** [libuv/libuv#3670] [Q&A]
Hi, everyone.

I am looking for the source code of specific development libuv version, but there is no 1.14.2 verion at the Github.

I need this to analyze suspicious (malicious?) software that contains this library (screenshot). Recompiled LIB and PDB files will be added to the disassembler.

![image](https://user-images.githubusercontent.com/91150176/177477471-dc0383ba-7b86-472e-9ed3-af5cb90b4665.png)

**Comment by bnoordhuis** [comment]
The author probably built it off the v1.x branch after the v1.14.1 release but before v1.15.0, so it could be any commit between b0f9fb2a07a5e638b1580fe9a42a356c3ab35f37...8b69ce1419d2958011d415a636810705c36c2cc2 (and any local changes the author made, of course.)
